canonical/anbox-cloud.io
# Update website to match latest version of the copy doc [WD-4935]

## Done

- Removed secondary navigation
- Added global nav
- Updated content according to the latest version of the copy doc - referenced in the copydoc block of the index.html file

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Open the latest copy doc file
- Check that the content of the website matches that of the copy doc


## Issue / Card

Fixes [WD-4935](https://warthogs.atlassian.net/browse/WD-4935)

[WD-4935]: https://warthogs.atlassian.net/browse/WD-4935?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
